aw-server-rust.service

```diff
@@ -17,8 +16,7 @@
 
 [Service]
 Type=simple
-ExecStart=/usr/local/bin/aw-server-rust
-WorkingDirectory=/usr/local/share/aw-server-rust/
+ExecStart=/opt/activitywatch/aw-server-rust
```

Comment:
This file needs to be built too at package time and the makefile's variable for DESTDIR needs to be used for the exec path.